project/Build.scala

```diff
@@ -258,14 +258,15 @@ object ScaldingBuild extends Build {
 
   lazy val scaldingParquet = module("parquet").settings(
     libraryDependencies ++= Seq(
-      "com.twitter" % "parquet-cascading" % "1.4.0",
+      "com.twitter" % "parquet-cascading" % parquetVersion,
+      "com.twitter" % "parquet-avro" % parquetVersion,
```

Comment:
We might end up with module creep in dependencies with lots of modules then
for all combinations. Could we maybe go the provided route to keep them
separate ? It's own form of uglyness... But avoids build file growing hugely

On Monday, February 9, 2015, Alex Levenson <notifications@github.com> wrote:

> In project/Build.scala
> <https://github.com/twitter/scalding/pull/1175#discussion_r24385556>:
>
> > @@ -258,14 +258,15 @@ object ScaldingBuild extends Build {
> >
> >    lazy val scaldingParquet = module("parquet").settings(
> >      libraryDependencies ++= Seq(
> > -      "com.twitter" % "parquet-cascading" % "1.4.0",
> > +      "com.twitter" % "parquet-cascading" % parquetVersion,
> > +      "com.twitter" % "parquet-avro" % parquetVersion,
>
> I think it would make more sense to split into a separate sub module per
> format, so that dependencies don't grow here.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/scalding/pull/1175/files#r24385556>.
>